pypolygames/utils/result.py

```diff
@@ -16,7 +16,7 @@ def parse_reward(reward):
         else:
             result["tie"] += 1
     result["total"] = len(reward)
-    result["avg"] = (sum(reward) / len(reward) + 1.) / 2.
+    result["avg"] = (sum(reward) / (len(reward) + 1.)) / 2.
```

Comment:
Not sure I understand. If I remembere correctly, the "+1)/2" stuff is here for moving from [-1,1] to [0,1].
The new line looks like something for getting rid of division by 0 ?
Maybe " ( (sum/(len+0.0001) ) + 1)/2".
Or maybe it's just me who misunderstood ?

Reply:
Yes, it is for getting rid of division by 0. Honestly the code just was like this in my local branch, so I assumed that's how it's supposed to be

Reply:
Why do we need to move from [-1,1] to [0,1] ?